tests/cmd/stability/main.go

```diff
@@ -234,16 +242,20 @@ func run() {
 		time.Sleep(3 * time.Minute)
 		fta.StartNodeOrDie(physicalNode, node)
 		oa.EmitEvent(nil, fmt.Sprintf("StartNode: %s on %s", node, physicalNode))
+		oa.WaitPodOnNodeReadyOrDie(deployedClusters, node)
 		oa.CheckRecoverOrDie(deployedClusters)
```

Comment:
After we restart the node, we should wait the pod running, then recover the failover.